contracts/Core/interface/IVoteManager.sol

```diff
@@ -8,20 +8,20 @@ interface IVoteManager {
 
     function reveal(
```

Comment:
It seems there are lot of definitions in interface that are not being used anywhere?
These should be removed to save space.

Reply:
@hrishikeshio yeah, Few getters are not being used.

Reply:
not just getters.
Where are we using reveal() ? many write functions are also not being used. please remove.